bin/utils/idam-lease-service-token.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -eu
+
```

Comment:
Is this file and bin/utils/idam-lease-user-token.sh exactly the same? If so can we not just amalgamate?

Reply:
idam-lease-user-token.sh isn't used so have removed the file